doc/tutorials/content/sources/executor/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
 project(executor)
 
-find_package(PCL 1.12 REQUIRED)
+find_package(PCL 1.12 COMPONENTS filters visualization REQUIRED)
```

Comment:
And common

Reply:
If you specify 1.12, the tutorial will fail to build. You need to specify 1.11.1.99 for the time being.

Reply:
1.11 works as well